sdks/java/core/src/test/java/org/apache/beam/sdk/util/BigQueryTableInserterTest.java

```diff
@@ -75,6 +77,7 @@
   @Rule public ExpectedLogs expectedLogs = ExpectedLogs.none(BigQueryTableInserter.class);
   @Mock private LowLevelHttpResponse response;
   private Bigquery bigquery;
+  private PipelineOptions options = PipelineOptionsFactory.create();
```

Comment:
Create a new instance in setUp() so that any changes to pipeline options isn't propagated between tests